### DISCUSSION TITLE
AWS import CloudFront Cache Policies or Origin Access Identity or other

### DISCUSSION THREAD

**Discussion by xnopre** [GoogleCloudPlatform/terraformer#917] [Q&A]
The tool is working great to import AWS CloudFront distributions. But is it possible to import AWS CloudFront Cache Policies or Origin Access Identity or other configurations ? 

Thanks

Xavier

**Comment by mmigliari** [comment]
Terraformer does not write the chace policies for cloudfront yet.  I also would like to request this feature, so that full CloudFront custom configs can be safely backed up.
